libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -369,6 +369,26 @@ bool AP_Arming::gps_checks(bool report)
             return false;
         }
     }
+
+    // check GPSs are within 50m of each other and that blending is healthy
+    if ((checks_to_perform & ARMING_CHECK_ALL) || (checks_to_perform & ARMING_CHECK_GPS_CONFIG)) {
+        float distance_m;
+        if (!gps.all_consistent(distance_m)) {
+            if (report) {
+                GCS_MAVLINK::send_statustext_all(MAV_SEVERITY_CRITICAL,
+                                                 "PreArm: GPSs inconsistent by %4.1fm",
+                                                  (double)distance_m);
+            }
+            return false;
+        }
+        if (!gps.blend_healthy()) {
```

Comment:
blend_health_check() or blend_ready_to_arm() would both be fine/better if you don't mind. My concern is that at an API level I expect blend_healthy() to be a way for external callers to assess if the GPS data is being blended. (I realize this is a weak argument in that nothing else does it yet).

If we init _blend_health_counter to 100 we force a period of healthy blending data which provides the protection I'm looking for on it.